Shoko.Server/Plex/Collection/SVR_PlexLibrary.cs

```diff
@@ -23,4 +23,13 @@ public Episode[] GetEpisodes()
             .DeserializeObject<MediaContainer<MediaContainer>>(data, Helper.SerializerSettings)
             .Container.Metadata;
     }
+
+    public Episode[] GetEpisode(string ratingKey)
+    {
+        var (_, data) = Helper.RequestFromPlexAsync($"/library/metadata/{ratingKey}").GetAwaiter()
+            .GetResult();
```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.getawaiter?view=net-8.0#remarks

generally speaking you shouldn't really be using `.GetAwaiter().GetResult()`